ai/Util/ai_command.py

```diff
@@ -1,9 +1,11 @@
 from RULEngine.Game.OurPlayer import OurPlayer
 from RULEngine.Util.Pose import Pose
-from RULEngine.Util.Position import Position
 from RULEngine.Util.SpeedPose import SpeedPose
 from enum import Enum
 
+from collections import ChainMap
+
+__all__ = ['AICommandType', 'AIControlLoopType', 'AICommand']
```

Comment:
__all__ encourages stuff like `from a import *`. Do we really need to add this?

Reply:
Will remove.